qiskit_nature/second_q/circuit/library/ansatzes/succd.py

```diff
@@ -77,12 +75,18 @@ def __init__(
                 the occupation of the spin orbitals. As such, the set of generalized excitations is
                 only determined from the number of spin orbitals and independent from the number of
                 particles.
-
+            mirror: boolean flag whether or not to include the symmetrically mirrored double
+                  excitations, while keeping the original number of circuit
+                  parameters. This results in mirrored excitations having identical parameter values.
+                  Enabling this parameter will result in the SUCCD ansatz referred to as
+                  "q-UCCSD0-full" in reference [1].
         Raises:
             QiskitNatureError: if the number of alpha and beta electrons is not equal.
         """
         self._validate_num_particles(num_particles)
         self._include_singles = include_singles
+        self._mirror = mirror
+        self._excitations_dict: Dict[str, List[Tuple[Tuple[int, ...], Tuple[int, ...]]]] = None
```

Comment:
```suggestion
        self._excitation_dict: Dict[str, List[Tuple[Tuple[int, ...], Tuple[int, ...]]]] = None
```
You are not being consistent with this variable name. Make sure this is the same everywhere